finagle-mysql/src/main/scala/com/twitter/finagle/mysql/Handshake.scala

```diff
@@ -89,6 +100,9 @@ object Handshake {
  *
  * @param charset default character established with the server.
  *
+ * @param enableFoundRows if the server should return the number
+ * of found (matched) rows, not the number of changed rows.
```

Comment:
add something like "for UPDATE and INSERT ... ON DUPLICATE KEY UPDATE type statements"